spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-cloudfoundry/src/main/java/org/springframework/cloud/data/module/deployer/cloudfoundry/CloudFoundryModuleDeployer.java

```diff
@@ -16,32 +16,89 @@
 
 package org.springframework.cloud.data.module.deployer.cloudfoundry;
 
+import java.util.HashMap;
 import java.util.Map;
 
+import org.springframework.cloud.data.core.ModuleDefinition;
 import org.springframework.cloud.data.core.ModuleDeploymentId;
 import org.springframework.cloud.data.core.ModuleDeploymentRequest;
 import org.springframework.cloud.data.module.ModuleStatus;
 import org.springframework.cloud.data.module.deployer.ModuleDeployer;
 
+/**
+ * A {@link ModuleDeployer} which deploys modules as applications running in a space in CloudFoundry.
+ *
+ * @author Paul Harris
+ * @author Steve Powell
+ */
 public class CloudFoundryModuleDeployer implements ModuleDeployer {
```

Comment:
I think (by looking at this) we should be naming this `AppModuleDeployer` (or something similar).  I'm assuming that we'll have two `ModuleDeployer` implementations similar to how we do with Lattice (one for processes and one for tasks) and the package/module indicates that it's for CF.